### PR TITLE
Simplify task project setup

### PR DESCRIPTION
**Pull request by Melonbwead**
This contains changes on how to get the task project setup and running, also updated the readme. :santa:

### CODE REVIEW

File: case/task/package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "Twitterized",
   "scripts": {
-      "start": "node app.js",
-      "watch": "gulp watch"
+      "start": "node app.js & npm run watch",
+      "watch": "./node_modules/.bin/gulp watch"
```
Comment (kimjoar) [inline case/task/package.json]: npm scripts should [already](https://docs.npmjs.com/cli/run-script) be looking at this location:

> In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts.

I think the only problem is that I still kept `npm install -g gulp` in the readme, even though it is not actually needed.

File: readme.md [hunk 2]
```diff
@@ -149,23 +149,19 @@ set up Twitter API access!
 ### Local setup
 
 Go to the `case/task` folder, then start by installing dependencies and starting
+
```
Comment (kimjoar) [inline readme.md]: newline typo?

**Comment by kimjoar** [comment]
Thanks for the contribution. Agree on `npm start` running both the server and watching.

File: readme.md (hunk 2 above)
Comment (Melonbwead) [inline readme.md]: Ye Thanks dude, was this at 3 am where you were :smile: ~

Comment (kimjoar) [inline readme.md]: Haha, yep, @adrianchen19888 — best time to look at Github issues ;)

**Pull request closed without merging**